lib/bigcommerce/prometheus/servers/thin/server.rb

```diff
@@ -31,6 +31,9 @@ def initialize(port: nil, host: nil, timeout: nil, logger: nil, thread_pool_size
             @rack_app = ::Bigcommerce::Prometheus::Servers::Thin::RackApp.new(timeout: timeout, logger: logger)
             super(@host, @port, @rack_app)
             ::Thin::Logging.logger = @logger
+            server_started = "[bigcommerce-prometheus] Prometheus server started on #{@host}:#{@port}"
+            @logger.info server_started
+            $stdout.puts server_started
```

Comment:
Why the double logging to `$stdout` and `@logger`?

Reply:
IMO we should keep it just to @logger; someone can configure this to use `Logger.new($stdout)` if they want stdout artifacts